### PR TITLE
#45 Update media links

### LINKED ISSUE #45: Review and update media in Shift_docs
At the moment, we were adding the name and the link to images to show them in the documentation:

`New Plug`

However, on the webpage the images in that format won't render. 
Image

Instead, we have to wrap the image reference in some HTML code to make it work. After some investigation, I figured out that the best way to do it is to:
- Have the images in the **same location** where the documentation file that uses them is (this is how it is structured in Adonis)
Image

- Use the tag figure and img from HTML to add the image. By using the figure tag, we will also have to add a figure caption. Below there's an example.
```
<figure>
      <img src="images/ui.png" alt="UI">
      <figcaption><b>Figure 1</b>: Shift UI at startup.</figcaption>
</figure>
```

By using this structure, we will be able to see the images rendered in Git Hub and the webpage, making it easier to review.

In this issue, we should update all the images that do not follow the right format.

### PR DESCRIPTION
**Pull request by juliavisserinbibo**
## Issue 
Closes #45 

## Description
Updates the links of media in pages that were early merged in the documentation. It also deletes unused images.

### CODE REVIEW

File: docs/getting_started/basics/authoring.md [hunk 1]
```diff
@@ -30,19 +37,31 @@ Certain nodes in Shift allow the creation of new plugs. To create a new plug, `R
...
 To remove a plug, `Right-Click` on the plug and select the "*Remove Plug*" option. This action will delete the plug and destroy all existing connections associated with it.
 
-![Removing Plug](../../images/remove_plugs.gif)
+<figure>
+      <img src="images/remove_plugs.gif" alt="Remove Plugs">
+      <figcaption><b>Figure 5</b>: Removing a connected plug from node.</figcaption>
+</figure>
+
 
 ## Removing a Connection
```
Comment (hectorjimenezinbibo) [inline docs/getting_started/basics/authoring.md:66]: Should we add a gif for removing a connection to be aligned with the previous sections?

File: docs/getting_started/basics/debug.md [hunk 2]
```diff
@@ -7,7 +7,10 @@ While working on a workflow, debugging may be necessary to ensure the graph exec
 
 When a node does not execute successfully it will turn red and the workflow execution will be interrupted. In such case, the node will store the traceback generated by the error. To inspect the traceback, `Right Click` on the node and select the *Traceback* option. This will pop up a dialog with the error traceback content. 
 
-![Traceback](../../images/traceback.gif)
+<figure>
+      <img src="images/traceback.gif" alt="Traceback">
```
Comment (hectorjimenezinbibo) [inline docs/getting_started/basics/debug.md:11]: In this gif still appears the old "Operator Name" naming in the inspector, should we update it?

File: docs/getting_started/basics/debug.md [hunk 3]
```diff
@@ -28,8 +31,10 @@ Shift provides many ways of executing workflows in a modular way. For instance,
 
 Additionally, Shift allows to add breakpoints to the nodes. When a node is flagged with a breakpoint, the execution of the graph will stop once it reaches that node. To add a breakpoint, `Right Click` on the node and select the *Breakpoint* option. To remove it, follow the same steps. 
 
-![Breakpoint](../../images/breakpoint.gif)
-
+<figure>
+      <img src="images/breakpoint.gif" alt="Breakpoint">
```
Comment (hectorjimenezinbibo) [inline docs/getting_started/basics/debug.md:35]: In this gif still appears the old "Operator Name" naming in the inspector, should we update it?

File: docs/getting_started/basics/execute.md [hunk 4]
```diff
@@ -4,32 +4,52 @@ The execution of a node in a Shift workflow consist in the evaluation of its int
...
 There are different ways a workflow can be executed, which depend on which nodes will get computed. 
 
-## ![Execute All Button](../../images/toolbar/execute_all_default.svg){:width="25" height="25"} Execute All
+## ![Execute All Button](images/execute_all_default.svg){style="width:5%"} Execute All
 
 The "Execute All" action will compute all the dirty nodes present in the workflow. 
 
-![Execute All](../../images/execute_all.gif)
+<figure>
+      <img src="images/execute_all.gif" alt="Execute All">
```
Comment (hectorjimenezinbibo) [inline docs/getting_started/basics/execute.md:12]: In this gif still appears the old "Operator Name" naming in the inspector, should we update it?

File: docs/getting_started/basics/execute.md [hunk 5]
```diff
@@ -4,32 +4,52 @@ The execution of a node in a Shift workflow consist in the evaluation of its int
...
 
-## ![Execute Selected Button](../../images/toolbar/execute_selected_default.svg){:width="25" height="25"} Execute Selected
+
+## ![Execute Selected Button](images/execute_selected_default.svg){style="width:5%"} Execute Selected
 
 The "Execute Selected" action will compute all the selected nodes in the workflow. To do so, Shift will identify the workflow sub-graph required to evaluate the desired nodes and execute it.
 
-![Execute Selected](../../images/execute_selected.gif)
+<figure>
+      <img src="images/execute_selected.gif" alt="Execute Selected">
```
Comment (hectorjimenezinbibo) [inline docs/getting_started/basics/execute.md:22]: In this gif still appears the old "Operator Name" naming in the inspector, should we update it?

File: docs/getting_started/basics/execute.md [hunk 6]
```diff
@@ -4,32 +4,52 @@ The execution of a node in a Shift workflow consist in the evaluation of its int
...
+
 
-## ![Execute Next Button](../../images/toolbar/execute_next_default.svg){:width="25" height="25"} Execute Next
+## ![Execute Next Button](images/execute_next_default.svg){style="width:5%"} Execute Next
 
 The "Execute Next" action will compute the next dirty node present in the workflow. This allows to execute one node at a time and becomes useful to closely follow and inspect the processing of the workflow.
 
-![Execute Next](../../images/execute_step_by_step.gif)
+<figure>
+      <img src="images/execute_step_by_step.gif" alt="Execute next">
```
Comment (hectorjimenezinbibo) [inline docs/getting_started/basics/execute.md:32]: In this gif still appears the old "Operator Name" naming in the inspector, should we update it?

File: docs/getting_started/basics/execute.md [hunk 7]
```diff
@@ -4,32 +4,52 @@ The execution of a node in a Shift workflow consist in the evaluation of its int
...
+
 
-## ![Clear Execution Button](../../images/toolbar/clear_all_default.svg){:width="25" height="25"} Clear Execution
+## ![Clear Execution Button](images/clear_all_default.svg){style="width:5%"} Clear Execution
 
 The "Clear Execution" action will set all the nodes in the workflow to dirty. All **connected** inputs and output values will be cleared to their default values. This forces Shift to recompute all nodes the next time the workflow has to be executed. 
 
-![Clear Execution](../../images/clear_execution.gif)
+<figure>
+      <img src="images/clear_execution.gif" alt="Clear execution">
```
Comment (hectorjimenezinbibo) [inline docs/getting_started/basics/execute.md:42]: In this gif still appears the old "Operator Name" naming in the inspector, should we update it?

File: docs/getting_started/basics/execute.md [hunk 8]
```diff
@@ -4,32 +4,52 @@ The execution of a node in a Shift workflow consist in the evaluation of its int
...
 
-## ![Live Execution Button](../../images/toolbar/execute_live_default.svg){:width="25" height="25"} Live Execution
+
+## ![Live Execution Button](images/execute_live_default.svg){style="width:5%"} Live Execution
 
 The "Live Execution" action allows to trigger the execution of the whole workflow whenever a change to any node is done. While the "Live Execution" is turned on, the other execution buttons are disabled. To re-enable all the execution buttons, turn Live Execution off.
 
-![Live Execution](../../images/live_execution.gif)
+<figure>
+      <img src="images/live_execution.gif" alt="Live Execution">
```
Comment (hectorjimenezinbibo) [inline docs/getting_started/basics/execute.md:52]: In this gif still appears the old "Operator Name" naming in the inspector, should we update it?

File: docs/getting_started/basics/nesting_workflows.md [hunk 9]
```diff
@@ -5,16 +5,21 @@ In Shift, any workflow can be embedded into a node and executed within another w
...
 
 The plugs from these nodes will act as data pathways, linking information set externally in the workflow to the current workflow and viceversa.
 
 ## Workflow Node
 
 The `Workflow` node allows to embed the execution of a workflow inside another workflow. This node has a *file* plug, where the workflow to nest is to be indicated. When the file plug content is set to reference a valid workflow file (with a *.sft* extension), the node will automatically update by creating the input and output plugs corresponding to the sourced workflow inputs and outputs. Then, when the `Workflow` node gets executed, it will execute the referenced workflow.
 
-![Nested Workflows](../../images/workflow_nesting.gif)
+<figure>
+      <img src="images/workflow_nesting.gif" alt="Workflow nesting">
```
Comment (hectorjimenezinbibo) [inline docs/getting_started/basics/nesting_workflows.md:20]: In this gif still appears the old "Operator Name" naming in the inspector, should we update it?

File: docs/getting_started/basics/authoring.md (hunk 1 above)
Comment (juliavisserinbibo) [reply docs/getting_started/basics/authoring.md:66]: I thought that removing a connection is such an easy thing that there's no need to add a gif

File: docs/getting_started/basics/debug.md (hunk 2 above)
Comment (juliavisserinbibo) [reply docs/getting_started/basics/debug.md:11]: I guess we have to update them all 😢

File: docs/getting_started/basics/nesting_workflows.md (hunk 9 above)
Comment (juliavisserinbibo) [reply docs/getting_started/basics/nesting_workflows.md:20]: Changed 
https://github.com/Inbibo/shift_docs/pull/61/commits/69e91498ee0d9a270c8014ac34740c78c4906515

File: docs/getting_started/basics/execute.md (hunk 8 above)
Comment (juliavisserinbibo) [reply docs/getting_started/basics/execute.md:52]: Changed 
https://github.com/Inbibo/shift_docs/pull/61/commits/69e91498ee0d9a270c8014ac34740c78c4906515

File: docs/getting_started/basics/execute.md (hunk 7 above)
Comment (juliavisserinbibo) [reply docs/getting_started/basics/execute.md:42]: Changed 
https://github.com/Inbibo/shift_docs/pull/61/commits/69e91498ee0d9a270c8014ac34740c78c4906515

File: docs/getting_started/basics/execute.md (hunk 6 above)
Comment (juliavisserinbibo) [reply docs/getting_started/basics/execute.md:32]: Changed 
https://github.com/Inbibo/shift_docs/pull/61/commits/69e91498ee0d9a270c8014ac34740c78c4906515

File: docs/getting_started/basics/execute.md (hunk 5 above)
Comment (juliavisserinbibo) [reply docs/getting_started/basics/execute.md:22]: Changed 
https://github.com/Inbibo/shift_docs/pull/61/commits/69e91498ee0d9a270c8014ac34740c78c4906515

File: docs/getting_started/basics/execute.md (hunk 4 above)
Comment (juliavisserinbibo) [reply docs/getting_started/basics/execute.md:12]: Changed 
https://github.com/Inbibo/shift_docs/pull/61/commits/69e91498ee0d9a270c8014ac34740c78c4906515

File: docs/getting_started/basics/debug.md (hunk 2 above)
Comment (juliavisserinbibo) [reply docs/getting_started/basics/debug.md:11]: Changed 
https://github.com/Inbibo/shift_docs/pull/61/commits/69e91498ee0d9a270c8014ac34740c78c4906515

Comment (hectorjimenezinbibo) [reply docs/getting_started/basics/debug.md:11]: I think this one has not been updated.

File: docs/getting_started/basics/debug.md (hunk 3 above)
Comment (hectorjimenezinbibo) [reply docs/getting_started/basics/debug.md:35]: I think this one has not been updated.

**Review by hectorjimenezinbibo — CHANGES_REQUESTED** [review]

File: docs/getting_started/basics/debug.md (hunk 2 above)
Comment (juliavisserinbibo) [reply docs/getting_started/basics/debug.md:11]: Now is updated

File: docs/getting_started/basics/debug.md (hunk 3 above)
Comment (juliavisserinbibo) [reply docs/getting_started/basics/debug.md:35]: Now is updated

**Review by hectorjimenezinbibo — APPROVED** [review]

**Review by leegeertseninbibo — APPROVED** [review]

**Review by albertochimentiinbibo — APPROVED** [review]

**Pull request merged**